<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EndRemontWindow</class>
 <widget class="QDialog" name="EndRemontWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Окончание ремонта</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    border: 1px solid #9F9F9F;
    font-size: 14px;
    font-weight: 800;
    color: black;
    border-radius: 4px;
    margin-top: 2.5ex; 
    padding-top: 2ex;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
}


QPushButton {
    border-radius: 3px;
    border: 1px solid #9F9F9F;
	padding-left: 8px;
	padding-right: 8px;
	padding-top: 4px;
	padding-bottom: 4px;
}

QPushButton:hover { 
	background-color: white;
}

QPushButton:pressed {
background-color: LightGray;
    position: relative;
     top: 4px; left: 4px;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>14</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Серийный №</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="leNumber"/>
       </item>
       <item>
        <widget class="QToolButton" name="tbNumber">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,1,1" columnminimumwidth="0,0,0,0">
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Серийный №</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="lbDevice">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QComboBox" name="cbReason"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Дата окончания</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Дефект</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Действия</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="3">
      <widget class="QLineEdit" name="leDefect"/>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QDateEdit" name="deDate">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::TimeSpec::LocalTime</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Рекламация</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLabel" name="lbName">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="lbNumber">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="3">
      <widget class="QLineEdit" name="leAction"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Причина</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Устройство</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLabel" name="lbClaim">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Комментарий</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Наименование</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="3">
      <widget class="QPlainTextEdit" name="ptRemark"/>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="QCheckBox" name="cbScrap">
       <property name="text">
        <string>Ремонт выполнить не удалось. Утилизировать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbEndRemont">
       <property name="text">
        <string>Закочить ремонт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>EndRemontWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
